mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -292,43 +324,244 @@ def run(self):  # {{{
         for regionIndex in regionIndicesToPlot:
             region = regions[regionIndex]
 
-            ohc = dsOHC.ohc.isel(nOceanRegionsTmp=regionIndex)
+            # Plot temperature, salinity and OHC anomalies (and full fields)
+            # trends with depth
+
+            #  First T vs depth/time
```

Comment:
@gstreletz: as you can see, I have moved up the anomaly calculations/plots because those are more relevant so it is nice to see them first. The `if plotOriginalFields` statements follow just below.